extensions/cornerstone/src/services/ViewportActionCornersService/ViewportActionCornersService.ts

```diff
@@ -33,34 +33,34 @@ class ViewportActionCornersService extends PubSubService {
 
   public setServiceImplementation({
     getState: getStateImplementation,
-    setComponent: setComponentImplementation,
-    setComponents: setComponentsImplementation,
-    clear: clearImplementation,
+    addComponent: addComponentImplementation,
+    addComponents: addComponentsImplementation,
+    clear: clearComponentsImplementation,
   }): void {
     if (getStateImplementation) {
       this.serviceImplementation._getState = getStateImplementation;
     }
-    if (setComponentImplementation) {
-      this.serviceImplementation._setComponent = setComponentImplementation;
+    if (addComponentImplementation) {
+      this.serviceImplementation._addComponent = addComponentImplementation;
```

Comment:
Since you are tweaking this anyways, can you just make it this.setComponent = setComponentImplementation and skip the extra level of indirection.  Having that doesn't really help anything, and not doing it means the service implementation can directly call this.X for any of the other services.

Reply:
Let's plan to clean this up in the next version